dist/restore/index.js

```diff
@@ -37903,9 +37903,17 @@ AbortError.prototype = Object.create(Error.prototype);
 AbortError.prototype.constructor = AbortError;
 AbortError.prototype.name = 'AbortError';
 
+const URL$1 = Url.URL || whatwgUrl.URL;
```

Comment:
Did node type update introduced these save and restore js file changes?

Reply:
No @bishal-pdMSFT . I suppose the .js files in main are out of date. I've updated those as well here by building those, as it was one of the tests.